dot/peerset/peerstate.go

```diff
@@ -57,8 +57,8 @@ type Info struct {
 
 // node represents state of a single node that we know about
 type node struct {
-	// list of Set the node belongs to.
-	// always has a fixed size equal to the one of PeersState Set. The various possible Set
+	// list of Set, the node belongs to.
```

Comment:
I'm a bit confused by this comment (before and after)... is `state` a list of sets the node belongs to?

Reply:
afaiu, yes

Reply:
please then change to:

```suggestion
	// state is a list of sets the node belongs to.
```

Reply:
changed